frontend/packages/console-dynamic-plugin-sdk/package.json

```diff
@@ -15,6 +15,9 @@
     "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'",
     "publish": "yarn build && yarn publish dist --no-git-tag-version"
   },
+  "dependencies": {
+    "react-virtualized": "9.x"
```

Comment:
Until now, we've put all the deps into `devDependencies` and pick/reflect relevant ones into the generated `package.json` file (this will change once we refactor the plugin SDK vs. plugin runtime pkgs). Until then, let's put this one into `devDependencies` as well.